net.codelogistics.webapps.json

```diff
@@ -0,0 +1,40 @@
+{
+    "app-id" : "net.codelogistics.webapps",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "webapps",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=xdg-data"
```

Comment:
why do you need this?

Reply:
it's not allowed by default.

Reply:
You should use this instead https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.portal.DynamicLauncher.html

Reply:
Thank you so much, the program now uses portals, so I have removed that line from the manifest.